### DISCUSSION TITLE
Hide Myself in the Overlay

### DISCUSSION THREAD

**Discussion by ghost** [overlayeddev/overlayed#176] [Q&A]
Is there a way to hide myself from the overlay? Or at least only show myself when I'm talking? It just looks cleaner imo. Thanks.

**Comment by Hacksore** [comment]
Currently there is not way to hide yourself but it's something we can add. This would need to be an opt-int flag so we are getting to the point where we need a new settings page.

There is also #166 which is experimenting with adding a new config library so that we can access config items in rust.

Here is what I am thinking we need to support this properly:

- https://github.com/overlayeddev/overlayed/pull/166
- https://github.com/overlayeddev/overlayed/issues/177

What we can do to make this a "beta" like feature is add it to the config now and you manually have to add it in.

So that'd be something like add a new default called `showOwnUser: true` and it can be added to you `config.json`.

**Comment by ghost** [comment]
Are you able to make an issue so I can give a shot at it?
